backend/layers/thirdparty/s3_provider.py

```diff
@@ -140,3 +140,19 @@ def list_directory(self, bucket_name: str, src_dir: str) -> Iterable[str]:
         paginator = self.client.get_paginator("list_objects_v2")
         for page in paginator.paginate(Bucket=bucket_name, Prefix=src_dir):
             yield from (content["Key"] for content in page.get("Contents", []))
+
+    def put_object(self, bucket_name: str, object_key: str, body: str) -> None:
+        """
+        Uploads a json object to S3
+        """
+        self.client.put_object(
+            Bucket=bucket_name,
+            Key=object_key,
+            Body=body,
+        )
+
+    def create_bucket(self, bucket_name: str, location: str) -> None:
+        """
```

Comment:
this was added to satiate `mock_s3` during testing - not actually creating any bucket